src/torchcodec/decoders/_core/VideoDecoder.cpp

```diff
@@ -877,7 +953,17 @@ void VideoDecoder::maybeSeekToBeforeDesiredPts() {
   streamInfo.discardFramesBeforePts = desiredPts;
 
   decodeStats_.numSeeksAttempted++;
-  if (canWeAvoidSeeking(desiredPts)) {
+
+  // TODO_CODE_QUALITY The different signature is unfortunate
```

Comment:
Why is the signature different? Don't we do the same computation in `canWeAvoidSeekingAudio()`?

Reply:
I think we could pass `desiredPtsSeconds_` to `canWeAvoidSeekingVideo`, that would lead to some minor redundancy but probably still cleaner